# DeepSMILE: Self-supervised heterogeneity-aware multiple instance learning for DNA damage response defect classification directly from H&E whole-slide images
> [!info]+ <center>Metadata</center>
> 
> |<div style="width: 5em">Key</div>|Value|
> |--:|:--|
> |æ–‡çŒ®ç±»å‹|journalArticle|
> |æ ‡é¢˜|DeepSMILE: Self-supervised heterogeneity-aware multiple instance learning for DNA damage response defect classification directly from H&E whole-slide images|
> |çŸ­æ ‡é¢˜|DeepSMILE|
> |ä½œè€…|[[Yoni Schirris]]ã€ [[Efstratios Gavves]]ã€ [[Iris Nederlof]]ã€ [[Hugo Mark Horlings]]ã€ [[Jonas Teuwen]]|
> |æœŸåˆŠåç§°|[[arXivï¼š2107.09405 [cs, eess]]]|
> |DOI|[](https://doi.org/)|
> |å­˜æ¡£ä½ç½®||
> |é¦†è—ç›®å½•||
> |ç´¢ä¹¦å·||
> |ç‰ˆæƒ||
> |åˆ†ç±»|[[00_Histology, 202205]]|
> |æ¡ç›®é“¾æ¥|[My Library](zotero://select/library/items/5FSA5QBY)|
> |PDF é™„ä»¶|[arXiv Fulltext PDF](zotero://open-pdf/library/items/KVLMGWAZ)|
> |å…³è”æ–‡çŒ®||
> ^Metadata


> [!example]- <center>æœ¬æ–‡æ ‡ç­¾</center>
> 
> `$=dv.current().file.tags`


> [!quote]- <center>Abstract</center>
> 
> We propose a Deep learning-based weak label learning method for analysing whole slide images (WSIs) of Hematoxylin and Eosin (H&E) stained tumorcells not requiring pixel-level or tile-level annotations using Self-supervised pre-training and heterogeneity-aware deep Multiple Instance LEarning (DeepSMILE). We apply DeepSMILE to the task of Homologous recombination deficiency (HRD) and microsatellite instability (MSI) prediction. We utilize contrastive self-supervised learning to pre-train a feature extractor on histopathology tiles of cancer tissue. Additionally, we use variability-aware deep multiple instance learning to learn the tile feature aggregation function while modeling tumor heterogeneity. Compared to state-of-the-art genomic label classification methods, DeepSMILE improves classification performance for HRD from $70.43\pm4.10\%$ to $83.79\pm1.25\%$ AUC and MSI from $78.56\pm6.24\%$ to $90.32\pm3.58\%$ AUC in a multi-center breast and colorectal cancer dataset, respectively. These improvements suggest we can improve genomic label classification performance without collecting larger datasets. In the future, this may reduce the need for expensive genome sequencing techniques, provide personalized therapy recommendations based on widely available WSIs of cancer tissue, and improve patient care with quicker treatment decisions - also in medical centers without access to genome sequencing resources.


> [!tldr]- <center>éšè—ä¿¡æ¯</center>
> 
> itemType:: journalArticle
> title:: DeepSMILE: Self-supervised heterogeneity-aware multiple instance learning for DNA damage response defect classification directly from H&E whole-slide images
> shortTitle:: DeepSMILE
> creators:: [[Yoni Schirris]]ã€ [[Efstratios Gavves]]ã€ [[Iris Nederlof]]ã€ [[Hugo Mark Horlings]]ã€ [[Jonas Teuwen]]
> publicationTitle:: [[arXivï¼š2107.09405 [cs, eess]]]
> journalAbbreviation:: 
> volume:: 
> issue:: 
> pages:: 
> language:: 
> DOI:: [](https://doi.org/)
> ISSN:: 
> url:: [http://arxiv.org/abs/2107.09405](http://arxiv.org/abs/2107.09405)
> archive:: 2022-07-25
> archiveLocation:: 
> libraryCatalog:: 
> callNumber:: 
> rights:: 
> extra:: ğŸ·ï¸ _empahsisã€_readã€/Doneã€Computer Science - Computer Vision and Pattern Recognitionã€Electrical Engineering and Systems Science - Image and Video Processingã€ngs
> collection:: [[00_Histology, 202205]]
> tags:: #_empahsis #Computer_Science_-_Computer_Vision_and_Pattern_Recognition #Electrical_Engineering_and_Systems_Science_-_Image_and_Video_Processing #ngs #Done 
> related:: 
> itemLink:: [My Library](zotero://select/library/items/5FSA5QBY)
> pdfLink:: [arXiv Fulltext PDF](zotero://open-pdf/library/items/KVLMGWAZ)
> qnkey:: 2021_Schirris_DeepSMILEï¼šSelf-super_KEY-5FSA5QBY
> date:: 2021-07-28
> dateY:: 2021
> dateAdded:: 2022-04-07
> dateModified:: 2022-07-26
> year:: 2022
> dateCurrent:: 2022-07-27
> time:: 21:03:51
> week:: æ˜ŸæœŸä¸‰
> yearMonth:: 2022-07
> dateWeek:: 2022-07-27 æ˜ŸæœŸä¸‰
> dateTime:: 2022-07-27 21:03:51
> dateWeekTime:: 2022-07-27 21:03:51 æ˜ŸæœŸä¸‰
> 
> abstract:: We propose a Deep learning-based weak label learning method for analysing whole slide images (WSIs) of Hematoxylin and Eosin (H&E) stained tumorcells not requiring pixel-level or tile-level annotations using Self-supervised pre-training and heterogeneity-aware deep Multiple Instance LEarning (DeepSMILE). We apply DeepSMILE to the task of Homologous recombination deficiency (HRD) and microsatellite instability (MSI) prediction. We utilize contrastive self-supervised learning to pre-train a feature extractor on histopathology tiles of cancer tissue. Additionally, we use variability-aware deep multiple instance learning to learn the tile feature aggregation function while modeling tumor heterogeneity. Compared to state-of-the-art genomic label classification methods, DeepSMILE improves classification performance for HRD from $70.43\pm4.10\%$ to $83.79\pm1.25\%$ AUC and MSI from $78.56\pm6.24\%$ to $90.32\pm3.58\%$ AUC in a multi-center breast and colorectal cancer dataset, respectively. These improvements suggest we can improve genomic label classification performance without collecting larger datasets. In the future, this may reduce the need for expensive genome sequencing techniques, provide personalized therapy recommendations based on widely available WSIs of cancer tissue, and improve patient care with quicker treatment decisions - also in medical centers without access to genome sequencing resources.


ğŸ‘£â¿ğŸ‘£


## âœï¸ ç¬”è®°åŒº

>[!inbox]- <center>ğŸ“« ç¬”è®°ç®€æŠ¥</center>
>
> â° importDate:: 2022-07-27

> [!IMPORTANT]+ <center>ğŸŒ± ç ”è¯»å°è±¡</center>  
>
>ğŸ“Œ comment::  

> [!WARNING]+ <center>ğŸ£ æ€»ç»“</center>  
>
>ğŸ¯ ç ”ç©¶é—®é¢˜::  
ğŸ” ç ”ç©¶èƒŒæ™¯::  
ğŸš€ ç ”ç©¶æ–¹æ³•::  
ğŸ” ç ”ç©¶æ€è·¯::  
ğŸ“º ä¸»è¦å†…å®¹::  
ğŸ‰ ç ”ç©¶ç»“è®º::  
ğŸ—ï¸ åˆ›æ–°ç‚¹::  
ğŸ’© ç ”ç©¶å±€é™::  
ğŸ¾ ç ”ç©¶å±•æœ›::  
âœï¸ å¤‡æ³¨::  


ğŸ‘£â¿ğŸ‘£

## ğŸ“ æ³¨é‡Šç¬”è®° KVLMGWAZ

> <span style="font-size: 15px;color: gray">ğŸ“ 2021-Schirris-DeepSMILE: Self-supervised heterogeneity-aware multiple instance learning for DNA damage response defect classification directly from H&amp;E whole-slide images</span>

^KEYrefTitle

> <span class="highlight" style="background-color: #ffd40040">multiple instance learning</span> ([p1](zotero://open-pdf/library/items/KVLMGWAZ?page=1&annotation=MD7Q37PY))

^KEYMD7Q37PY

> <span class="highlight" style="background-color: #ffd40040">not requiring pixel-level or tile-level annotations</span> ([p1](zotero://open-pdf/library/items/KVLMGWAZ?page=1&annotation=A8FVAGXA))

^KEYA8FVAGXA

> <span class="highlight" style="background-color: #ff666640">We apply DeepSMILE to the task of Homologous recombination deficiency (HRD) and microsatellite instability (MSI) prediction.</span> ([p1](zotero://open-pdf/library/items/KVLMGWAZ?page=1&annotation=DND7ZG86))

^KEYDND7ZG86

> <span class="highlight" style="background-color: #ff666640">contrastive self-supervised learning</span>  
> backbone ([p1](zotero://open-pdf/library/items/KVLMGWAZ?page=1&annotation=5QLTJPUL))

^KEY5QLTJPUL

> <span class="highlight" style="background-color: #ffd40040">These improvements suggest we can improve genomic label classification performance without collecting larger datasets.</span> ([p1](zotero://open-pdf/library/items/KVLMGWAZ?page=1&annotation=47CYU8I7))

^KEY47CYU8I7

> <span class="highlight" style="background-color: #ff666640">In the future, this may reduce the need for expensive genome sequencing tech</span> ([p1](zotero://open-pdf/library/items/KVLMGWAZ?page=1&annotation=VSJ8I3UB))

^KEYVSJ8I3UB

> <span class="highlight" style="background-color: #ffd40040">DNA Damage Response (DDR)</span> ([p1](zotero://open-pdf/library/items/KVLMGWAZ?page=1&annotation=U4BBJQ58))

^KEYU4BBJQ58

> <span class="highlight" style="background-color: #ffd40040">Our proposed method, DeepSMILE, is a two-stage weakly supervised method applied to HRD and MSI classification.</span> ([p4](zotero://open-pdf/library/items/KVLMGWAZ?page=4&annotation=FAD6M4GG))

^KEYFAD6M4GG

> <span class="highlight" style="background-color: #ffd40040">3.1.1. Colorectal cancer tissue dataset: TCGA-CRCk</span> ([p4](zotero://open-pdf/library/items/KVLMGWAZ?page=4&annotation=8MV6PPYB))

^KEY8MV6PPYB

> <span class="highlight" style="background-color: #a28ae540">We use the pre-processed colorectal tumor tiles extracted from Formalin-Fixed, Paraffin-Embedded (FFPE) WSIs from the Cancer Genome Atlas (TCGA) with accompanying binarized MSI labels from Kather (2019).</span>  
> æ•°æ® ([p4](zotero://open-pdf/library/items/KVLMGWAZ?page=4&annotation=RY2F3U63))

^KEYRY2F3U63

> <span class="highlight" style="background-color: #ffd40040">The number of tiles per WSI ranges from 6 to 4,064 with an average of 375 tiles per WSI.</span> ([p4](zotero://open-pdf/library/items/KVLMGWAZ?page=4&annotation=B8878F2V))

^KEYB8878F2V

> <span class="highlight" style="background-color: #ff666640">feature variability-aware deep multiple instance learning.</span> ([p6](zotero://open-pdf/library/items/KVLMGWAZ?page=6&annotation=ER5GKY5W))

^KEYER5GKY5W

> <span class="highlight" style="background-color: #ff666640">tile-level MLP classifier</span> ([p6](zotero://open-pdf/library/items/KVLMGWAZ?page=6&annotation=XSSWWN9T))

^KEYXSSWWN9T

> <span class="highlight" style="background-color: #ff666640">loss = WeightedCE(y, pred)</span> ([p6](zotero://open-pdf/library/items/KVLMGWAZ?page=6&annotation=3A4MANM4))

^KEY3A4MANM4

> <span class="highlight" style="background-color: #ff666640">normalized temperaturescaled cross-entropy (NT-Xent) loss</span> ([p7](zotero://open-pdf/library/items/KVLMGWAZ?page=7&annotation=VN8IPQD2))

^KEYVN8IPQD2

> <span class="highlight" style="background-color: #ff666640">VarMIL</span> ([p8](zotero://open-pdf/library/items/KVLMGWAZ?page=8&annotation=JNX9GWUZ))

^KEYJNX9GWUZ

> <span class="highlight" style="background-color: #ff666640">We compare with two baseline models: DeepMIL and a tilesupervision baseline with a tile-level majority vote.</span> ([p8](zotero://open-pdf/library/items/KVLMGWAZ?page=8&annotation=S8UTNQNC))

^KEYS8UTNQNC

> <span class="highlight" style="background-color: #ff666640">In these models, the patient-level label is assigned to each tile from the tumor bed, and the classification task is defined to predict the WSI-level label directly from each individual tile. The final WSI-level prediction is defined as a variant of the majority vote, computed as the proportion of positively labeled tiles</span>  
> baseline: æ¯ä¸ªtileéƒ½åˆ†é…ä¸€ä¸ªslideæ ‡ç­¾ï¼Œæœ€åé€šè¿‡å°‘æ•°æœä»å¤šæ•°å†èµ‹äºˆslideæ ‡ç­¾ ([p8](zotero://open-pdf/library/items/KVLMGWAZ?page=8&annotation=8XCRU6ZG))

^KEY8XCRU6ZG

> <span class="highlight" style="background-color: #ff666640">4.2.2. Weak label classification: DeepMIL</span> ([p8](zotero://open-pdf/library/items/KVLMGWAZ?page=8&annotation=3FS6CV7J))

^KEY3FS6CV7J

> <span class="highlight" style="background-color: #ffd40040">On TCGA-BC we train for 16 epochs with a batch size of 16 on a subsample of 500 tiles per WSI and evaluate the performance every 5 iterations</span> ([p9](zotero://open-pdf/library/items/KVLMGWAZ?page=9&annotation=VJ3G9VKJ))

^KEYVJ3G9VKJ

> <span class="highlight" style="background-color: #ffd40040">. For TCGA-CRCk we train for 16 epochs on all tiles with a batch size of 16,</span> ([p9](zotero://open-pdf/library/items/KVLMGWAZ?page=9&annotation=PIABK7DK))

^KEYPIABK7DK

> <span class="highlight" style="background-color: #ffd40040">Weak label classification with VarMIL</span> ([p9](zotero://open-pdf/library/items/KVLMGWAZ?page=9&annotation=URI4NV42))

^KEYURI4NV42







