# Artificial intelligence to identify genetic alterations in conventional histopathology
> [!info]+ <center>Metadata</center>
> 
> |<div style="width: 5em">Key</div>|Value|
> |--:|:--|
> |æ–‡çŒ®ç±»å‹|journalArticle|
> |æ ‡é¢˜|Artificial intelligence to identify genetic alterations in conventional histopathology|
> |çŸ­æ ‡é¢˜||
> |ä½œè€…|[[Didem Cifci]]ã€ [[Sebastian Foersch]]ã€ [[Jakob Nikolas Kather]]|
> |æœŸåˆŠåç§°|[[The Journal of Pathology]]|
> |DOI|[10.1002/path.5898](https://doi.org/10.1002/path.5898)|
> |å­˜æ¡£ä½ç½®||
> |é¦†è—ç›®å½•|9.883|
> |ç´¢ä¹¦å·|1|
> |ç‰ˆæƒ||
> |åˆ†ç±»|[[202205, 00_Histology]]|
> |æ¡ç›®é“¾æ¥|[My Library](zotero://select/library/items/YUSPDRLF)|
> |PDF é™„ä»¶|[Cifci ç­‰ã€‚ - 2022 - Artificial intelligence to identify genetic altera.pdf](zotero://open-pdf/library/items/3C3AJXZE)|
> |å…³è”æ–‡çŒ®||
> ^Metadata


> [!example]- <center>æœ¬æ–‡æ ‡ç­¾</center>
> 
> `$=dv.current().file.tags`


> [!quote]- <center>Abstract</center>
> 
> Precision oncology relies on the identiï¬cation of targetable molecular alterations in tumor tissues. In many tumor types, a limited set of molecular tests is currently part of standard diagnostic workï¬‚ows. However, universal testing for all targetable alterations, especially rare ones, is limited by the cost and availability of molecular assays. From 2017 to 2021, multiple studies have shown that artiï¬cial intelligence (AI) methods can predict the probability of speciï¬c genetic alterations directly from conventional hematoxylin and eosin (H&E) tissue slides. Although these methods are currently less accurate than gold standard testing (e.g. immunohistochemistry, polymerase chain reaction or next-generation sequencing), they could be used as pre-screening tools to reduce the workload of genetic analyses. In this systematic literature review, we summarize the state of the art in predicting molecular alterations from H&E using AI. We found that AI methods perform reasonably well across multiple tumor types, although few algorithms have been broadly validated. In addition, we found that genetic alterations in FGFR, IDH, PIK3CA, BRAF, TP53, and DNA repair pathways are predictable from H&E in multiple tumor types, while many other genetic alterations have rarely been investigated or were only poorly predictable. Finally, we discuss the next steps for the implementation of AI-based surrogate tests in diagnostic workï¬‚ows.


> [!tldr]- <center>éšè—ä¿¡æ¯</center>
> 
> itemType:: journalArticle
> title:: Artificial intelligence to identify genetic alterations in conventional histopathology
> shortTitle:: 
> creators:: [[Didem Cifci]]ã€ [[Sebastian Foersch]]ã€ [[Jakob Nikolas Kather]]
> publicationTitle:: [[The Journal of Pathology]]
> journalAbbreviation:: The Journal of Pathology
> volume:: 
> issue:: 
> pages:: path.5898
> language:: en
> DOI:: [10.1002/path.5898](https://doi.org/10.1002/path.5898)
> ISSN:: 0022-3417, 1096-9896
> url:: [https://onlinelibrary.wiley.com/doi/10.1002/path.5898](https://onlinelibrary.wiley.com/doi/10.1002/path.5898)
> archive:: 2022-07-25
> archiveLocation:: 
> libraryCatalog:: 9.883
> callNumber:: 1
> rights:: 
> extra:: ğŸ·ï¸ _empahsisã€_readã€/Doneã€review
> collection:: [[202205, 00_Histology]]
> tags:: #_empahsis #Done #review
> related:: 
> itemLink:: [My Library](zotero://select/library/items/YUSPDRLF)
> pdfLink:: [Cifci ç­‰ã€‚ - 2022 - Artificial intelligence to identify genetic altera.pdf](zotero://open-pdf/library/items/3C3AJXZE)
> qnkey:: 2022_Cifci_Artificial intellige_KEY-YUSPDRLF
> date:: 2022-04-21
> dateY:: 2022
> dateAdded:: 2022-05-22
> dateModified:: 2022-07-26
> year:: 2022
> dateCurrent:: 2022-07-27
> time:: 21:04:00
> week:: æ˜ŸæœŸä¸‰
> yearMonth:: 2022-07
> dateWeek:: 2022-07-27 æ˜ŸæœŸä¸‰
> dateTime:: 2022-07-27 21:04:00
> dateWeekTime:: 2022-07-27 21:04:00 æ˜ŸæœŸä¸‰
> 
> abstract:: Precision oncology relies on the identiï¬cation of targetable molecular alterations in tumor tissues. In many tumor types, a limited set of molecular tests is currently part of standard diagnostic workï¬‚ows. However, universal testing for all targetable alterations, especially rare ones, is limited by the cost and availability of molecular assays. From 2017 to 2021, multiple studies have shown that artiï¬cial intelligence (AI) methods can predict the probability of speciï¬c genetic alterations directly from conventional hematoxylin and eosin (H&E) tissue slides. Although these methods are currently less accurate than gold standard testing (e.g. immunohistochemistry, polymerase chain reaction or next-generation sequencing), they could be used as pre-screening tools to reduce the workload of genetic analyses. In this systematic literature review, we summarize the state of the art in predicting molecular alterations from H&E using AI. We found that AI methods perform reasonably well across multiple tumor types, although few algorithms have been broadly validated. In addition, we found that genetic alterations in FGFR, IDH, PIK3CA, BRAF, TP53, and DNA repair pathways are predictable from H&E in multiple tumor types, while many other genetic alterations have rarely been investigated or were only poorly predictable. Finally, we discuss the next steps for the implementation of AI-based surrogate tests in diagnostic workï¬‚ows.


ğŸ‘£â¿ğŸ‘£


## âœï¸ ç¬”è®°åŒº

>[!inbox]- <center>ğŸ“« ç¬”è®°ç®€æŠ¥</center>
>
> â° importDate:: 2022-07-27

> [!IMPORTANT]+ <center>ğŸŒ± ç ”è¯»å°è±¡</center>  
>
>ğŸ“Œ comment::  

> [!WARNING]+ <center>ğŸ£ æ€»ç»“</center>  
>
>ğŸ¯ ç ”ç©¶é—®é¢˜::  
ğŸ” ç ”ç©¶èƒŒæ™¯::  
ğŸš€ ç ”ç©¶æ–¹æ³•::  
ğŸ” ç ”ç©¶æ€è·¯::  
ğŸ“º ä¸»è¦å†…å®¹::  
ğŸ‰ ç ”ç©¶ç»“è®º::  
ğŸ—ï¸ åˆ›æ–°ç‚¹::  
ğŸ’© ç ”ç©¶å±€é™::  
ğŸ¾ ç ”ç©¶å±•æœ›::  
âœï¸ å¤‡æ³¨::  


ğŸ‘£â¿ğŸ‘£

## ğŸ“ æ³¨é‡Šç¬”è®° 3C3AJXZE

> <span style="font-size: 15px;color: gray">ğŸ“ 2022-Cifci-Artificial intelligence to identify genetic alterations in conventional histopathology</span>

^KEYrefTitle

> <span class="highlight" style="background-color: #ff666640">Prediction of mutations from H&amp;E slides</span> ([p2](zotero://open-pdf/library/items/3C3AJXZE?page=2&annotation=YGC6FMX8))

^KEYYGC6FMX8

> <span class="highlight" style="background-color: #ff666640">Zeng et al developed a method that detects BRCA1 and BRCA2 mutations with AUROC values of 0.95 and 0.91, respectively, on the held-out dataset in high-grade serous ovarian carcinoma and also showed that integrating genomics, transcriptomics, and proteomics data with the image features leads to better prognostic models compared with images alone [80].</span> ([p6](zotero://open-pdf/library/items/3C3AJXZE?page=6&annotation=8X3VLBTU))

^KEY8X3VLBTU

> <span class="highlight" style="background-color: #ff666640">Owing to the fact that TMB tumors are inflamed, and possibly due to other morphological changes, multiple studies have shown that it is possible to infer TMB from H&amp;E histology [43,45,73,74,79].</span> ([p7](zotero://open-pdf/library/items/3C3AJXZE?page=7&annotation=D2ZNT7D4))

^KEYD2ZNT7D4

> <span class="highlight" style="background-color: #ff666640">[73].</span>  
> tmbå›å½’ ([p7](zotero://open-pdf/library/items/3C3AJXZE?page=7&annotation=Q5D6ZNI4))

^KEYQ5D6ZNI4

> <span class="highlight" style="background-color: #ff666640">All studies reviewed here, however, lack an external test dataset, which is a prerequisite for moving towards clinical application [94].</span>  
> ç¼ºä¹å¤–éƒ¨æ•°æ®é›†éªŒè¯ ([p7](zotero://open-pdf/library/items/3C3AJXZE?page=7&annotation=D6ARY3JT))

^KEYD6ARY3JT

> <span class="highlight" style="background-color: #ff666640">In ovarian cancer, the predictive power of DL models is comparatively much higher, as they reached AUROC values of 0.92 in the held-out test datasets, where the sample size was 114 with the model fed with a training dataset of 115 patients [80]. Newer technologies such as multiple instance learning (MIL) and self-supervised learning (SSL) have improved classification performance by more than 10% compared with the conventional WSI classification model with a sample size as low as 100 patients in colorectal cancer [57]. Schrris et al showed that the performance of the MSI classifier can reach above 0.90 in colorectal cancer by combining SSL with an attention-based DL model [57].</span> ([p8](zotero://open-pdf/library/items/3C3AJXZE?page=8&annotation=PRF57QJ7))

^KEYPRF57QJ7







